tools/env_version_update.py

```diff
@@ -37,13 +37,14 @@ def main(dir_to_scan):
     for item in os.listdir(dir_to_scan):
         item_abs_path = os.path.abspath(os.path.join(dir_to_scan, item))
         if os.path.isdir(item_abs_path):
-            env_info_json = os.path.join(item_abs_path, ENV_INFO_JSON)
-            with open(env_info_json) as json_file:
-                metadata = json.load(json_file)
-                metadata["environmentVersionId"] = str(ObjectId())
-            with open(env_info_json, "w") as json_file:
-                json.dump(metadata, json_file, indent=2)
-                json_file.write("\n")
+            if not env or env == item:
```

Comment:
How about moving this condition right after the for loop at line 37 and do the following
```
    if env and env != item
        continue
```